Emby.Server.Implementations/LiveTv/TunerHosts/HdHomerun/HdHomerunUdpStream.cs

```diff
@@ -121,7 +121,7 @@ public override async Task Open(CancellationToken openCancellationToken)
             //OpenedMediaSource.Path = tempFile;
             //OpenedMediaSource.ReadAtNativeFramerate = true;
 
-            MediaSource.Path = _appHost.GetLocalApiUrl("127.0.0.1", true) + "/LiveTv/LiveStreamFiles/" + UniqueId + "/stream.ts";
+            MediaSource.Path = _appHost.GetLocalApiUrl("127.0.0.1") + "/LiveTv/LiveStreamFiles/" + UniqueId + "/stream.ts";
```

Comment:
I would like to disagree with this change... I made it for a reason.

Reply:
I mean, I've added it so LiveTV would always use `http://127.0.0.1:8096` regardless of https settings, why do you remove this?

Reply:
My understanding was that the original bug was that LiveTV only couldn't use HTTPS if HTTPS was being provided by a reverse proxy. Was that incorrect? If the server itself is listening with HTTPS can LiveTV use it?

Reply:
I would say it depends. What if user provided a self-signed certificate? `ffmpeg` could refuse to connect to it. Plus connecting over ssl to a localhost seems _just a little bit_ too paranoid (and wasteful of CPU cycles).

Reply:
Okay, yeah that makes sense.

What if the server is not listening on HTTP at all? I was [discussing](https://github.com/jellyfin/jellyfin-web/pull/1140#issuecomment-620746402) that with @dkanada in the web PR. It seems like with this in mind it might make sense to continue forcing the server to listen on HTTP and not adding that toggle?

Reply:
Right now it's impossible to turn off server listening on HTTP. I think we can still make that toggle turn off HTTP on anything but localhost, so `http://localhost:8096` is always available for `ffmpeg`.

Reply:
> I think we can still make that toggle turn off HTTP on anything but localhost, so `http://localhost:8096` is always available for `ffmpeg`.

Good idea 👍

Okay, I'll look into getting the forceHttp option back for this method (whenever I get the chance to stop doing code reviews 😅). I think I'll refactor the method a bit as well to use `UriBuilder` instead of `StringBuilder`

Reply:
Okay, finally got around to updating this to add back the `forceHttps` functionality. I changed it up a bit to use a separate method instead of passing a boolean flag, which seems a bit cleaner to me, let me know what you think

Reply:
Yeah, I like this much more. It adds more responsibility to the caller (to pass a correct pair of options instead of a simple flag), but this is hidden by `GetLoopbackUrl()` which I actually like. Should have done so myself :)